src/app/cli/default_cli_delegate.cpp

```diff
@@ -143,6 +145,8 @@ void DefaultCliDelegate::exportFiles(Context* ctx, DocExporter& exporter)
 int DefaultCliDelegate::execScript(const std::string& filename,
                                    const Params& params)
 {
+  ScriptInputChain scriptInputChain;
+  App::instance()->inputChain().prioritize(&scriptInputChain, nullptr);
```

Comment:
I'm not 100% sure, probably we should do this only when `-batch` was specified (?)

Remember that we can specify something like `aseprite.exe file.aseprite -script script.lua`

You can test if the script is executed with the UI/Editor/DocView available in that case (and using the `ScriptInputChain` might broke that case where the `DocView` can handle the `InputChain`).